database/infrastructure/construct.py

```diff
@@ -196,6 +196,8 @@ def __init__(
             database = aws_rds.DatabaseInstance(self, **database_config)
 
         hostname = database.instance_endpoint.hostname
+        self.db_security_group = database.connections.security_groups[0]
```

Comment:
we don't need to make the change in this pr but I think at this point we should add a `self.database=database` so that the rds construct in it's entirety is available to other constructs in the deployment (there are a bunch of other database properties that we have added to the construct over time, too)